tcms/rpc/tests/test_testexecution.py

```diff
@@ -274,6 +274,60 @@ def test_get_links(self):
         self.assertEqual(logs[0]['url'], 'http://kiwitcms.org')
 
 
+class TestExecutionHistory(APITestCase):
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.execution = TestExecutionFactory()
+
+    def test_history_for_non_existing_execution(self):
+        with self.assertRaisesRegex(XmlRPCFault, "TestExecution matching query does not exist."):
+            self.rpc_client.TestExecution.history(-5)
+
+    def test_history_new_execution(self):
+        execution = TestExecutionFactory()
+
+        history = self.rpc_client.TestExecution.history(execution.pk)
+
+        self.assertEqual(1, len(history))
+
+    def test_history(self):
+        build = BuildFactory()
+        self.rpc_client.TestExecution.update(self.execution.pk, {
+            "build": build.pk,
+        })
+
+        user = UserFactory()
+        self.rpc_client.TestExecution.update(self.execution.pk, {
+            "assignee": user.pk,
+        })
+        self.rpc_client.TestExecution.update(self.execution.pk, {
+            "tested_by": user.pk,
+        })
```

Comment:
it is better to update the test object directly, not through the API b/c it is a bit faster.